src/transformStateWithClones.js

```diff
@@ -7,7 +7,47 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const options = Object.values(actions);
+  const result = [];
+
+  for (const option of options) {
+    const { keysToRemove, type } = option;
+
+    if (type === 'clear') {
+      result.push({});
+    }
+
+    if (type === 'addProperties') {
```

Comment:
Use [`switch case`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) in cases where you constantly compare one value with others
Fix all cases